docs/jwst/cube_build/arguments.rst

```diff
@@ -44,26 +46,25 @@ each band will be created.
   with the options above [band, channel, grating, filter] to fully control the type of IFU
   cubes to make.
 
-  - ``output_type = band`` is the default mode for calspec3 and creates IFU cubes containing only one band
+  - ``output_type = band`` creates IFU cubes containing only one band
     (channel/sub-channel for MIRI or grating/filter combination for NIRSpec).
 
   - ``output_type = channel`` creates a single IFU cube from each unique channel of MIRI data
-    (or just those channels set by the 'channel' option).
+    (or just those channels set by the 'channel' option). The is the default mode for the calspec3 pipeline for
+    MIRI data. 
 
   - ``output_type = grating`` combines all the gratings in the NIRSpec data or set by the
-    grating option into a single IFU cube.
+    grating option into a single IFU cube. The is the default mode for the calspec3 pipeline for
+    NIRSpec data. 
 
   - ``output_type = multi`` combines data  into a single "uber" IFU cube, this the default mode for calspec2. If in addition,
     channel, band, grating, or filter are also set, then only the data set by those
     parameters will be combined into an "uber" cube.
 
 The following arguments control the size and sampling characteristics of the output IFU cube.
 
-``scale1``
-  The output cube's spaxel size in axis 1 (spatial).
-
-``scale2``
-  The output cube's spaxel size in axis 2 (spatial).
+``scale12``
```

Comment:
Because there's now only one "scale" argument that relates to the spatial scale I'm wondering if we should change the name to something more generic, like just "scale" or perhaps "scales" (to go with "scalew"). The "scale12" name seems kind of contrived. It doesn't really fit with the "scalew" argument name.

Reply:
Ok I thought about 'scales' - but then I was not sure if it would be understood that is scale for spatial axes.
scale_spatial ?

Reply:
I could live with `scale_spatial`, but then we should also change `scalew` to `scale_wave` for consistency. Or will changing these param names break too much existing code/scripts/notebooks?

Reply:
What about scalexy?  Cubes generally have an x/y/wavelength axis, and scalexy is neither as ambiguous as scales or as contrived as scale12.

Reply:
Ooh, I like it!  David for the win.

Reply:
I don't see an update here to change to `scalexy`.

Reply:
Ok I missed that one. Thanks